Fix loading button being larger than intended in some Windows browsers. (#194)
StenaIT/stenajs-webui: CHANGELOG.md

`Banner` component now has `success` & `warning` variants.

### Bug fixes

`DateTextInput` popover content is now wrapped with a popupRef.
This fixes a bug where `useMultiOnClickOutside` would cause the popover to close before being able trigger a change.

> `PrimaryButton` is now not larger than intended in some Windows browsers.

Reviewer: ```suggestion
`PrimaryButton` is no longer larger than intended on some Windows browsers.
```
